deckhand/objects/base.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2017 AT&T Intellectual Property.  All other rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import datetime
+
+from oslo_versionedobjects import base
```

Comment:
It is a known danger with OVO that for documents with multi-line data that are whitespace sensitive, OVO will break them. Because Deckhand won't know which Kinds of documents this is, it seems like OVO requires us to Base64 encode everything to ensure the document sent to Deckhand is the document it returns. This breaks all manner of querying.